google/cloud/advisorynotifications/CMakeLists.txt

```diff
@@ -0,0 +1,213 @@
+# ~~~
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ~~~
+
+include(GoogleapisConfig)
+set(DOXYGEN_PROJECT_NAME "Advisory Notifications API C++ Client")
+set(DOXYGEN_PROJECT_BRIEF
+    "A C++ Client Library for the Advisory Notifications API")
+set(DOXYGEN_PROJECT_NUMBER "${PROJECT_VERSION}")
+set(DOXYGEN_EXCLUDE_SYMBOLS "internal")
+set(DOXYGEN_EXAMPLE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/quickstart)
+
+unset(mocks_globs)
+unset(source_globs)
+set(service_dirs "v1/")
+foreach (dir IN LISTS service_dirs)
+    string(REPLACE "/" "_" ns "${dir}")
+    list(APPEND source_globs "${dir}*.h" "${dir}*.cc" "${dir}internal/*")
+    list(APPEND mocks_globs "${dir}mocks/*.h")
+    list(APPEND DOXYGEN_EXAMPLE_PATH
+         "${CMAKE_CURRENT_SOURCE_DIR}/${dir}samples")
+    list(APPEND DOXYGEN_EXCLUDE_SYMBOLS "advisorynotifications_${ns}internal")
+endforeach ()
+
+include(GoogleCloudCppDoxygen)
+google_cloud_cpp_doxygen_targets("advisorynotifications" DEPENDS
+                                 google-cloud-cpp::advisorynotifications_protos)
```

Comment:
This is missing `cloud-docs`:

https://github.com/googleapis/google-cloud-cpp/blob/e1de7b77130910965b1cf247f891743e6f07873f/google/cloud/accessapproval/CMakeLists.txt#L39

Probably a bug I introduced in the scaffold generator.